pyfolio/tests/test_txn.py

```diff
@@ -41,25 +36,15 @@ def test_get_turnover(self):
         result = get_turnover(positions, transactions)
         assert_series_equal(result, expected)
 
-        # Monthly freq
-        index = date_range('01-01-2015', freq='M', periods=1)
-        expected = Series([0.0], index=index)
-        result = get_turnover(positions, transactions, period='M')
-        assert_series_equal(result, expected)
-
-        transactions = DataFrame(data=[[1, 1, 10, 'A']]*len(dates),
+        transactions = DataFrame(data=[[1, 1, 10, 'A']]*len(dates) +
+                                 [[2, -1, 10, 'B']]*len(dates),
                                  columns=['sid', 'amount', 'price', 'symbol'],
-                                 index=dates)
+                                 index=dates.append(dates)).sort_index()
 
-        expected = Series([0.5]*len(dates), index=dates)
+        expected = Series([4.0] + [2.0] * (len(dates) - 1), index=dates)
```

Comment:
Could you add a brief comment explaining the 4.0 here? (I know why it is, just for future readers)